lib/check-tags.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const log = require('./out')({});
+
+function checkTags(app, mod, name) {
+  // Returns true if the module should be skipped
+
+  if ((app.excludeTags && !mod.tags) || (!app.withTags && !app.excludeTags)) {
+    return false; // No checks to run.
+  } else if (app.withTags && !mod.tags) {
+    return true; // No tags for this module
+  }
+
+  if (typeof mod.tags === 'string') {
```

Comment:
this is to do with the lookup tags rather than than the args. So you could have `"tags": "test"` or `"tags": ["test1", "test2"]`